wifiphisher/pywifiphisher.py

```diff
@@ -925,6 +925,9 @@ def run():
 
     time.sleep(3)
 
+    #no longer need mac_matcher
+    mac_matcher.unbind()
```

Comment:
Thinking of it again, isn't it much simpler if we just delete the `mac_matcher` object when we are done with it?